src/style.css

```diff
@@ -1 +1,42 @@
-/* add styles here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+  background-color: #fff;
+  width: max-content;
```

Comment:
Don't fix container size (if there is no such a requirement). Setting `width: max-content;` on the body element might cause layout issues. It's better to let the content size dictate the width of the body. Consider removing this property unless there is a specific requirement for it.